apps/openmw/mwmechanics/aisequence.cpp

```diff
@@ -366,6 +372,18 @@ void AiSequence::readState(const ESM::AiSequence::AiSequence &sequence)
     if (!sequence.mPackages.empty())
         clear();
 
+    // If there is more than one non-combat, non-pursue package in the list, enable repeating.
+    for (std::vector<ESM::AiSequence::AiPackageContainer>::const_iterator it = sequence.mPackages.begin();
+         it != sequence.mPackages.end(); ++it)
```

Comment:
Looks like something's gone amiss after the last few changes. After loading a saved game AI travels aren't always repeating. Probably related to changing the AIPackage's getrepeat to default to off.